common/src/main/java/io/armory/plugin/observability/model/PluginMetricsDatadogConfig.java

```diff
@@ -0,0 +1,32 @@
+package io.armory.plugin.observability.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PluginMetricsDatadogConfig extends BaseMetricsIntegrationConfig {
+    // Datadog Specific Settings
+    @Builder.Default private boolean enabled = false;
```

Comment:
@Builder.default is not needed here as the @builder is on the class level?

Reply:
All other config classes have it. I tried to delete it and then build is not successful